### DISCUSSION TITLE
Exploring GPS Data Analysis: Using PacketSender as a Receptive Server for Cellular Network Tramas

### DISCUSSION THREAD

**Discussion by EasyConnectJC** [dannagle/PacketSender#329] [General]
Hello,

I'm interested in finding an application that allows me to analyze GPS device data sent over the cellular network using a SIM card with a public APN. These GPS devices can be configured to send information to a server of my choice using UDP or TCP protocols and a specific port.

I would like to know if PacketSender is a viable option to act as the receiving server for the GPS device data. My idea is to configure my notebook with the Portable Version for Windows to carry out this task, so I'd appreciate knowing if this is feasible.

I would be grateful for any information or recommendations you can provide on this matter.

Best regards,

**Comment by dannagle** [comment]
Packet Sender can certainly receive the packets, but it would not know what to do with them. The data would get dropped into the logs and then that's it. No analysis. The listener side of Packet Sender is better used as a troubleshooting tool as you figure out what is wrong with your more permanent service.

Packet Sender can work great as your client for sending.
